src/store/index.ts

```diff
@@ -317,6 +321,17 @@ const useTaxesStore = defineStore({
       }
       updateUrlQuery({ expenses: this.expenses });
     },
+    setExpensesManual(value: number) {
+      this.expensesAuto = false;
+      this.setExpenses(value);
+    },
+    setExpensesAuto(v: any) {
```

Comment:
any reason for this `v` parameter? It's not used and typescript is complaining about it in `Dashboard.vue`

Reply:
oops, forgot to cleanup that